src/finetune_model.py

```diff
@@ -81,35 +81,51 @@ def prepare_train_features(examples):
 
 
 tokenizer = AutoTokenizer.from_pretrained(cf.setting["model_checkpoint"])
+model = AutoModelForQuestionAnswering.from_pretrained(cf.setting["model_checkpoint"])
 
-dataset = create_dataset()
+# data = load_data(sampling=True)
+dataset = create_dataset(sampling=True)
 
 tokenized_datasets = dataset.map(prepare_train_features, batched=True, remove_columns=dataset["train"].column_names)
-print(tokenized_datasets["train"][0])
+# print(tokenized_datasets["train"][0])
 
 # Fine-tuning the model
-model = AutoModelForQuestionAnswering.from_pretrained(cf.setting["model_checkpoint"])
 
-args = TrainingArguments(
-    f"test-covid",
+training_args = TrainingArguments(
+    f"../models/test-covid",
     evaluation_strategy="epoch",
     learning_rate=2e-5,
     per_device_train_batch_size=cf.setting["batch_size"],
     per_device_eval_batch_size=cf.setting["batch_size"],
     num_train_epochs=3,
     weight_decay=0.01,
+
 )
 
+
+
+# training_args = TFTrainingArguments(
```

Comment:
I know you have another ticket for this, but this can benefit from some code cleaning